frontend/codecept.conf.js

```diff
@@ -0,0 +1,33 @@
+const LOCAL_LAUNCH_URL = 'http://localhost:3001';
+
+exports.config = {
+  output: './tests/acceptance/output',
+  helpers: {
+    Puppeteer: {
+      url: LOCAL_LAUNCH_URL,
+      show: true,
+      windowSize: '1200x900',
+    },
+  },
+  include: {
+    I: './tests/steps_file.js',
+    Page: './tests/acceptance/pages/*.js',
+  },
+  mocha: {},
+  bootstrap: null,
+  teardown: null,
+  hooks: [],
+  gherkin: {
+    features: './tests/acceptance/features/*/*.feature',
+    steps: ['./tests/acceptance/step_definitions/loginContext.js'],
```

Comment:
```suggestion
    steps: ['./tests/acceptance/step_definitions/*.js'],
```
There could be more contexts files in future!

Reply:
This is an array, so we can't do that. I tried but it didn't work.